Simplified/Book/UI/NYPLBookDetailView+Accessibility.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  NYPLBookDetailView+Accessibility.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-07-21.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
```

Comment:
this should probably be UIKit since you're using UIAccessibility. Maybe we get away with `Foundation` because we might be importing UIKit in the precompiled headers.